### ISSUE TITLE
QR Scanning not working on iOS

### ISSUE THREAD

**Issue by u56237z** [bitwarden/android#2148]
### Steps To Reproduce

try to scan a TOTP QR code

### Expected Result

generate TOTP

### Actual Result

nothing happens, the camera keeps showing the QR, it is slightly out of focus, the QR code is not recognized, there is no option to change camera settings. The default iPhone camera reads the same QR without any problem.

### Additional Context

just installed bitwarden on a brand new iPhone 14 Plus

### Operating System

iOS

### Operating System Version

16.0.3

### Device

iPhone 14 Plus2

### Build Version

2022.10.0 (2347)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by dbosompem** [comment]
Hi @u56237z can you try again to confirm if issue still exist? We've been unable to reproduce this so wanted to seek confirmation from your side as well.

**Comment by skinnybeans** [comment]
I can confirm that I am also having this problem on an iPhone 14 pro max with the same version of bitwarden.

The specific QR code I had problems with was github

**Comment by lukeify** [comment]
I am likewise experiencing this issue. The camera opens, but never properly focuses and as such cannot identify the QR code. iPhone 15 Pro running Bitwarden 2023.10.0 (5357).

**Comment by ericswpark** [comment]
Same issue here, iPhone 14 Pro Max. Weirdly it does not switch to the close-up wide angle lens, which is why I'm guessing it won't scan. A lot of apps had the same issue when the 14 Pro series first came out, until they got updated to support automatic lens switching. At this point, the app won't scan any small QR codes which sucks.

**Comment by alapitz** [comment]
This still reproduces for me iPhone 15 Pro

**Comment by bvinnerd** [comment]
Is this issue only occurring when the QR code is on a dark background? (this happens to me; I have to use the JS console in my browser to change background colors to white (#fff)

**Comment by u56237z** [comment]
idk if it applies to every QR on a dark background, but I can confirm it still doesnt work when trying to read Paperspace QR to enable its 2FA
________________________________

**Comment by glitchcrab** [comment]
I was also seeing issues with this on a 14 Pro Max with the latest Bitwarden version. I seem to have gotten it to work by expanding the blank area around the QR code with a white background and then slowly moving the phone backwards and forwards (and sometimes tilting it up and down slightly). With a little bit of patience the blue rectangle usually turns green and the code is scanned successfully. 

It's a pain though and definitely less than ideal.

**Comment by DoubleMCA** [comment]
I am also having this issue on and iPhone 14 Pro. I have recently switched from Raivo Authenticator to Ente Authenticator and as I am updating some seeds in Ente I wanted to add them to Bitwarden. I have two iOS devices, and iPhone 14 Pro and and iPad pro 12" - No matter which device I tried using to display the QR code from Ente and which I tried scanning to code to Bitwarden - none of them worked. The built in iOS password feature reads the QR codes fine as does the new Bitwarden Authentication app.
However the "proper" bitwarden app cannot scan the code. It just keeps trying to scan no matter what.
